charts/falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.9.0
```

Comment:
``
```suggestion
## 0.9.1
```
I know it's not following the semver, but we upgrade the minor at each new version of sidekick or ui, and the patch for this kind of change